alicloud/resource_alicloud_polardb_cluster_test.go

```diff
@@ -0,0 +1,123 @@
+package alicloud
+
+import (
+	"fmt"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/polardb"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+	"testing"
+)
+
+func TestAccAlicloudPolarDBCluster(t *testing.T) {
+	var v *polardb.DescribeDBClusterAttributeResponse
+	var ips []map[string]interface{}
+	rand := acctest.RandInt()
+	name := fmt.Sprintf("tf-testacc%sdnsrecordbasic%v.abc", defaultRegionToTest, rand)
+	resourceId := "alicloud_polardb_cluster.default"
+	var basicMap = map[string]string{
+		"description":   CHECKSET,
+		"db_node_class": CHECKSET,
+		"vswitch_id":    CHECKSET,
+		"db_type":       CHECKSET,
+		"db_version":    CHECKSET,
+	}
+	ra := resourceAttrInit(resourceId, basicMap)
+	serviceFunc := func() interface{} {
+		return &PolarDBService{testAccProvider.Meta().(*connectivity.AliyunClient)}
+	}
+	rc := resourceCheckInitWithDescribeMethod(resourceId, &v, serviceFunc, "DescribePolarDBClusterAttribute")
+	rac := resourceAttrCheckInit(rc, ra)
+
+	testAccCheck := rac.resourceAttrMapUpdateSet()
+	testAccConfig := resourceTestAccConfigFunc(resourceId, name, resourcePolarDBClusterConfigDependence)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+
+		// module name
+		IDRefreshName: resourceId,
+
+		Providers:    testAccProviders,
+		CheckDestroy: rac.checkResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccConfig(map[string]interface{}{
+					"db_type":             "MySQL",
+					"db_version":          "8.0",
+					"cluster_charge_type": "Postpaid",
+					"db_node_class":       "polar.mysql.x4.large",
+					"vswitch_id":          "${alicloud_vswitch.default.id}",
+					"description":         "${var.name}",
+				}),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck(nil),
+				),
+			},
+			{
+				ResourceName:      resourceId,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{"auto_renew", "auto_renew_period", "cluster_network_type",
+					"cluster_charge_type", "effective_time", "period", "renewal_status", "db_node_class"},
```

Comment:
I think the ignored parameters should be removed and them can be gotten.

Reply:
They are all input, but the query is very troublesome, to check several interfaces, are you sure need to find ？